mysql-test/suite/perfschema/r/crash_tokudb.result

```diff
@@ -0,0 +1,32 @@
+SELECT COUNT(*) > 0 FROM performance_schema.setup_consumers
```

Comment:
I don't think perfschema suite is the best location for these tests. Consider starting tokudb.perfschema